src/components/shared/WithSize.js

```diff
@@ -37,17 +37,34 @@ export function withSize<
 > {
   return class WithSizeWrapper extends React.PureComponent<*, State> {
     _resizeListener: () => void;
+    _visibilitychangeListener: () => void;
+    _isSizeInfoDirty: boolean = false;
     state = { width: 0, height: 0 };
 
     componentDidMount() {
       const container = findDOMNode(this); // eslint-disable-line react/no-find-dom-node
       if (!container) {
         throw new Error('Unable to find the DOMNode');
       }
+
       this._resizeListener = () => {
-        this._updateWidth(container);
+        if (!document.hidden) {
```

Comment:
This is a small change, but I generally find if there is an `if` / `else` branch, it's confusing to negate the condition. It is simpler to reason with it if were written like:

```js
if (document.hidden) {
  this._isSizeInfoDirty = true;
} else {
  this._updateWidth(container);
} 
```